ivy_tests/test_ivy/test_frontends/test_onnx/test_elementwise.py

```diff
@@ -0,0 +1,227 @@
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+
+# @handle_frontend_test(
+#     fn_tree="onnx.acos",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_acos(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.acosh",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_acosh(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.add",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("numeric"),
+#         num_arrays=2,
+#         large_abs_safety_factor=2.5,
+#         small_abs_safety_factor=2.5,
+#         safety_factor_scale="log",
+#     ),
+#     alpha=st.integers(min_value=1, max_value=5),
+# )
+# def test_onnx_add(
+#     *,
+#     dtype_and_x,
+#     alpha,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         rtol=1e-03,
+#         input=x[0],
+#         other=x[1],
+#         alpha=alpha,
+#     )
+#
+# @handle_frontend_test(
+#     fn_tree="onnx.asin",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+# )
+# def test_onnx_asin(
+#     *,
+#     dtype_and_x,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         input=x[0],
+#     )
+
+
+def test_onnx_abs_v2():
+    x = ivy.random_uniform(low=-100.0, high=0, shape=10)
+
+    onnx_abs = onnx.abs(x)
+    torch_abs = torch.abs(x)
+    jax_abs = jax.lax.abs(x)
+    paddle_abs = paddle.abs(x)
+    tf_abs = tf.abs(x)
+
+    all_list = [onnx_abs, jax_abs, paddle_abs, tf_abs]
+
+    assert torch.allclose(torch_abs, all_list)
```

Comment:
There is no point in comparing the results of different frontends. What we want is to check that the onnx frontend's returns setting different backends is the same as the return of the original onnx function.